docs/challenge/index.rst

```diff
@@ -0,0 +1,108 @@
+
+Lead organizers: Mathilde Papillon, Dr. Tegan Emerson, Dr. Henry Kvinge, Dr. Tim Doster, Dr. Bastian Rieck, Dr. Sophia Sanborn and Dr. Nina Miolane.
+
+
+Description of the Challenge
+-------------
+
+The purpose of this challenge is to foster reproducible research in Topological Deep Learning by crowdsourcing the open-source implementation of neural networks on topological domains. Participants are asked to contribute code for a previously existing Topological Neural Network (TNN) and train it on a provided protein interaction dataset. 
+
+Implementations are built using  `TopoModelX <https://github.com/pyt-team/TopoModelX/tree/main/topomodelx>`_, a Python package for deep learning on topological domains. Each submission takes the form of a  `Pull Request <https://github.com/pyt-team/TopoModelX/pulls>`_ to TopoModelX containing the necessary code for implementing a TNN from the literature. The implementation leverages the coding infrastructure and building blocks from TopoModelX.
+
+*Note:* *We invite participants to review this file regularly, as details are added when questions are submitted to the organizers.*
+
+⭐️ Publication Outcomes for Participants ⭐️
+-------------
+🎉 **Every submission respecting the submission requirements** will be included in a white paper summarizing findings of the challenge, published through the  `2023 TAG in Machine Learning Workshop <https://www.tagds.com/events/conference-workshops/tag-ml23>`_ at ICML. All participants with qualifying submissions will have the opportunity to co-author this publication.
+
+📘 Participants with the top 8 best submissions will have the additional opportunity to co-author a software paper on TopoModelX submitted to the **Journal of Machine Learning Research**.
+
+🏆 The best submissions in each topological domain (hypergraph, simplicial, cellular, combinatorial) will receive special recognition at the ICML TAG in Machine Learning Workshop.
+
+Deadline
+-------------
+The final Pull Request submission date and time must take place before **July 13, 2023 at 16:59 (Pacific Standard Time)**.
+Participants are welcome to modify their Pull Request until this time.
+
+How to Submit
+-------------
+Everyone can participate and participation is free. It is sufficient to:
+
+- send a Pull Request
+- respect Submission Requirements
+
+An acceptable Pull Request automatically subscribes a participant/team to the challenge.
+
+Guidelines
+-------------
+We encourage the participants to start submitting their Pull Request early on. This helps debug failing tests and helps address potential issues with the code.
+
+In the case of multiple submissions implementing the same model, earlier Pull Requests will be given priority consideration.
+
+Teams are accepted and there is no restriction on the number of team members.
+
+There is no restriction on the amount of submissions (Pull Requests) per participant/team. One model per Pull Request.
```

Comment:
I would change the order here and separate things.

1) one model per pull request
2) there is no restrictions on the number of..

Final question could be: can there be multiple implementations of the same model?

Reply:
will do!
Wrt your final question: Yes, although we specify that in that situation, models with earlier pull requests are given priority consideration (in the event of a tie for example). We hope that offering one winner/domain will encourage people to spread submissions across domains

Reply:
I think you should add earlier submission with the *same quality* get priority, otherwise you could just block all models by submitting a pull request for all of them initially -- so you had the earliest submission on all... ;)

Reply:
good idea!